src/tutorial/counter-app/single-function.md

```diff
@@ -132,6 +132,45 @@ to render a widget, i.e. a `Paragraph` widget.
     })?;
 ```
 
+````admonish note
+The most important thing to know with Ratatui is that you have to use one and _only_ one
```

Comment:
I think this info needs to be shorter in the tutorial and point at the rendering concepts page instead of going much beyond the "only call this once" advice.

Reply:
I shorten the text here and moved the additional information to an FAQ section.